### DISCUSSION TITLE
Disable (systems of) Plugins

### DISCUSSION THREAD

**Discussion by DasLixou** [bevyengine/bevy#6744] [Q&A] [answered]
Is there a way to disable plugins or just disable the systems of plugins? and then later reenable them? Thanks :D

**Comment by alice-i-cecile** [comment, marked as answer]
See the discussion in #2160. The TL;DR is no, unless you rebuild the plugin yourself.

This is one of the major long-term goals of the "stageless" rearchitecture: allowing users to apply configuration (in this case, run criteria) to plugins that they don't own without badly breaking internal logic and privacy.

**Reply by DasLixou** [reply]
the stageless rearchitecture sounds pretty cool. can't wait to see it. i already saw iyes_loopless, but is there also an official start of the new stageless rearchitecture to help and contribute? @alice-i-cecile

**Reply by alice-i-cecile** [reply]
Yes! https://github.com/bevyengine/bevy/pull/6587
